commands/scanpullrequest_test.go

```diff
@@ -18,23 +21,30 @@ import (
 	clitool "github.com/urfave/cli/v2"
 )
 
+const (
+	testMultiDirProjConfigPath = "testdata/config/frogbot-config-multi-dir-test-proj.yaml"
+	testProjSubdirConfigPath   = "testdata/config/frogbot-config-test-proj-subdir.yaml"
+	testCleanProjConfigPath    = "testdata/config/frogbot-config-clean-test-proj.yaml"
+	testProjConfigPath         = "testdata/config/frogbot-config-test-proj.yaml"
```

Comment:
For consistency reasons, let's change the extension of all the above files from `yaml` to `yml`.

Reply:
done